### PR TITLE
Update letters.json

### PR DESCRIPTION
**Pull request by old-df38257622**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by triach-rold — CHANGES_REQUESTED** [review]
Nice website. You might want to make another email if you're not comfortable making your email public because I'm pretty sure it's a required field. Anyways, reachable and complete.

**Review by triach-rold — COMMENTED** [review]
Nice view-counter, btw.

File: domains/letters.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
     "owner": {
         "username": "keli5",
-        "email": "kel5isgod@gmail.com"
+        "discord": "keli5_letters",
+        "email": "prefer not to provide",
```
Comment (CuteDog5695) [inline domains/letters.json:5]: if you dont want to provide your email but have added other socials then leave it blank

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by keli5** [comment]
added email

**Review by triach-rold — APPROVED** [review]
Looks good to me.

**Review by iostpa — APPROVED** [review]
seems good to me now
